Gemfile

```diff
@@ -1,5 +1,7 @@
 source "https://rubygems.org"
 
+# CircleCI images fail without this
+gem 'rb-readline'
```

Comment:
https://github.com/fastlane/fastlane/issues/21794#issuecomment-2021331335

Reply:
maybe let's add the link as a comment? that way if in the future we're wondering whether it's okay to remove we check the link

Reply:
good idea